mne/channels/layout.py

```diff
@@ -898,10 +898,10 @@ def read_montage(kind, ch_names=None, path=None, scale=True):
     return Montage(pos=pos, ch_names=ch_names_, kind=kind, selection=selection)
 
 
-def apply_montage(info, montage):
-    """Apply montage to EEG data.
+def apply_montage(info, montage, electrode_type='eeg'):
```

Comment:
we you the naming ch_type everywhere else. Please rename to ch_type unless someone objects.